src/main/java/firebase/sdk/dashboard/api/ReleaseResource.java

```diff
@@ -121,6 +121,12 @@ public Response enrollSDKinRelease(@PathParam("releaseName") String releaseName,
     }
 
     Release release = RELEASEDAO.getRelease(platform, releaseName);
+    long currentTime = System.currentTimeMillis() / 1000;
```

Comment:
Since you have added dependency injection, you could use https://docs.oracle.com/javase/8/docs/api/java/time/Instant.html#now-java.time.Clock-,
to make the code testable.

Feel free to add a TODO if you don't have the time to finish this :).